spec/cdata_spec.js

```diff
@@ -292,4 +292,33 @@ describe("XMLParser", function() {
         //console.log(JSON.stringify(result,null,4));
         expect(result).toEqual(expected);
     });
+
+    it("should parse XML with DOCTYPE without internal DTD", function() {
+        const xmlData = "<?xml version='1.0' standalone='no'?><!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\" \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\" ><svg><metadata>test</metadata></svg>";
+        const expected = {
+            "metadata": "test"
+        };
+
+        const result = parser.parse(xmlData, {
+            ignoreAttributes:       false,
+            allowBooleanAttributes: true
+        });
+        expect(result).toEqual(expected);
+    });
+
+    it("should parse XML with dashed elements, when - second symbol", function() {
```

Comment:
This is already covered. You can just update the description of existing test.

Reply:
Removed